src/myr/reference.js

```diff
@@ -168,57 +168,126 @@ let transformations = [
 
 let animations = [
   {
-    name: <span>spin(<span className="string">element</span>, <span className="number">magnitude</span>, <span className="bool">loop</span>, <span className="number">duration</span>)</span>,
+    name: <span>setLoop(<span className="bool">loop</span>)</span>,
+    description: <span>Updates the loop attribute in the cursor. Default: true</span>
+  },
+  {
+    name: <span>setDuration(<span className="number">duration</span>)</span>,
+    description: <span>Updates the duration attribute in the cursor in milliseconds. Default: 1000</span>
+  },
+  {
+    name: <span>setMagnitude(<span className="number">magnitude</span>)</span>,
+    description: <span>Updates the magnitude attribute in the cursor, in degrees and general units. Default: 1</span>
+  },
+  {
+    name: <span>spin(<span className="string">element</span>)</span>,
     description: <span>Spins the element around the y axis <span className="number">magnitude</span> degrees.</span>
   },
   {
-    name: <span>yoyo(<span className="string">element</span>, <span className="number">magnitude</span>, <span className="bool">loop</span>, <span className="number">duration</span>)</span>,
+    name: <span>yoyo(<span className="string">element</span>)</span>,
     description: <span>Bounces the element <span className="number">magnitude</span> units in a positive direction on the y axis for <span className="number">duration</span> milliseconds.</span>
```

Comment:
These descriptions should be updated to reflect that the values are no longer in the function but from the cursor.